src/pytsql/tsql.py

```diff
@@ -154,7 +154,7 @@ def _split(code: str, isolate_top_level_statements: bool = True) -> List[str]:
             " and is preventing the use of the faster C++ parser."
         )
 
-    logging.info("Started SQL script parsing")
+    logging.debug("Started SQL script parsing")
```

Comment:
All `logging.*` statements should most probably be turned into `logger.*`, where `logger = logging.getLogger("pytsql")`